### PR TITLE
Update compatibility with Python 3.11 beta 1

### PR DESCRIPTION
**Pull request by frenzymadness**
What has been added in 3.11 alpha 1: https://github.com/python/cpython/commit/98eee94421dcb42c15f2d7fc4cd21357722fbe2a
is now removed in beta 1: https://github.com/python/cpython/commit/944fffee8916cb94321fa33cd3a43f4108717746

### CODE REVIEW

**Comment by frenzymadness** [comment]
We have to wait for 3.11 beta 1 in fedora-python-tox image.

**Comment by frenzymadness** [comment]
Well, the arm job for py3[678] took five and half hours but they are all green, woohoo.

**Comment by frenzymadness** [comment]
I'm gonna test it without parallelisation, for the record:
* amd - 39m
* arm1 - 5h 35m
* arm2 - 4h 23
* ppc - 2h 38m
* s390x - 2h 10m

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+name: Run Tox tests
+
+jobs:
+  tox_test:
+    name: Tox test
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
```
Comment (hroncok) [inline .github/workflows/main.yml:18]:
```suggestion
      uses: docker/setup-qemu-action@v2
        platforms: amd64,arm64,ppc64le,s390x
```

**Comment by frenzymadness** [comment]
The last run has:
- amd: 1h 10m
- ppc: 3h 46m
- s390: 3h 58m
- both arm jobs have been canceled after 6 hours which is the maximum I guess.

So, to run the check in parallel is necessary and it might make sense to split arm jobs to three instead of two.

File: .github/workflows/main.yml (hunk 1 above)
Comment (frenzymadness) [reply .github/workflows/main.yml:18]: I know it's generally better to be more specific but what is the additional value in this? This step takes usually around 2 to 4 seconds.

**Comment by frenzymadness** [comment]
So now, the longest-running are Pythons 3.6 and 3.7 on ARM64 with around 3 hours and 40 minutes. That's not that bad and it's also far from the 6 hours limit.

File: .github/workflows/main.yml (hunk 1 above)
Comment (hroncok) [reply .github/workflows/main.yml:18]: > This step takes usually around 2 to 4 seconds.

I haven't known. Disregard then.

**Comment by frenzymadness** [comment]
@hroncok wanna do the final review?

**Comment by hroncok** [comment]
Will do.

**Review by hroncok — APPROVED** [review]
Every commit here looks good. CI passes.  Code changes and CI config are sane. Consider bumping the version.

**Pull request merged**